testing/test_config.py

```diff
@@ -46,6 +46,21 @@ def test_getcfg_empty_path(self):
         """correctly handle zero length arguments (a la pytest '')"""
         getcfg([""])
 
+    def test_setupcfg_uses_toolpytest_with_pytest(self, testdir):
+        p1 = testdir.makepyfile("def test(): pass")
+        testdir.makefile(
+            ".cfg",
+            setup="""
+                [tool:pytest]
+                testpaths=%s
```

Comment:
This test failed on Windows because using `%s` here will introduce `\` characters, resulting in an invalid path.

One way to fix it is to use `repr(str(p1))`, but decided to go with `p1.basename` because we don't need the full path in `testpaths` anyway.

Reply:
Thanks for fixing it!